monkey/infection_monkey/network/network_scanner.py

```diff
@@ -68,49 +97,39 @@ def get_victim_machines(self, max_find=5, stop_callback=None):
         :param stop_callback: A callback to check at any point if we should stop scanning
         :return: yields a sequence of VictimHost instances
         """
-
-        TCPscan = TcpScanner()
-        Pinger = PingScanner()
+        pool = Pool()
```

Comment:
Here you are using cpu_count as the pool size. However, network scanning is a heavy-IO process; wouldn't it be prudent to utilize the CPUs while we're waiting? I think I'd go with `Pool(cpu_count() * 4)`. Either way we choose to go, this is an important and non-trivial decision - please throw a line of documentation about it for future spelunkers of this file :)